frontend/src/organisms/SignUpForm.js

```diff
@@ -38,18 +38,10 @@ const schema = yup.object().shape({
       (value) => value > 0,
     )
     .label('Weight'),
-  gender: yup.string().required().oneOf(['male', 'female']).label('Gender'),
+  sex: yup.string().required().oneOf(['male', 'female']).label('Gender'),
   birthdate: yup.date().required().label('Birth date'),
   email: yup.string().email().required().label('Email'),
-  password: yup
-    .string()
-    .required()
-    .label('Password')
-    .test(
-      'Has min lenght?',
-      'ERROR: Minimum password length is 8 characters!',
-      (value) => value.length >= 8
-    ),
+  password: yup.string().required().label('Password'),
```

Comment:
pls vrať tam tu validaci jinak nám to dan hodí na hlavu že to není hotový

Reply:
Ta validace uplne nefunguje, kdyz to tam necham, tak to po napsani cehokoliv do inputu hodi error.

Reply:
Vraceno